### DISCUSSION TITLE
ReaInsert VST into Ildaeil Cardinal module & CV port

### DISCUSSION THREAD

**Discussion by subderground** [DISTRHO/Cardinal#710] [General]
Hello,
Cardinal rack is a little jewel, it works very well :-) Thanks a lot  !
Today, I tried to insert the ReaInsert VST  via Ildaeil host module in Cardinal Rack. It doesn't appears in the vst2 list, so I have tried  to open it with "Load from file", but nothing loads.
Have I missed something ?

Is this "8CV inputs expander for parameters (Ildaeil, Carla)" by @Simon-L is available with this last 24.09 version of Cardinal ?
Thanks.
Many greetz.
T.

**Comment by dromer** [comment]
As you can see the "8CV inputs expander" is still marked as a WIP/Work In Progress -> https://github.com/DISTRHO/Cardinal/pull/88
As such it has not been merged so not available in any release.

Only after a PR has been merged will you see features appear in any potential release.

**Reply by subderground** [reply]
Ok sorry, I thought the "WIP" was just before the last release. Thanks.

For the ReaInsert VST into Ildaeil Cardinal module, if it worked, it could powerfully integrate hardware modular stuffs inside Cardinal.    Nb: Sadly, Reainsert isn't JSFX.

**Reply by dromer** [reply]
It hasn't been worked on since late 2021, as you can clearly see from the history.

I have no idea what ReaInsert is, maybe share a link to a resource that explains it?

**Comment by dromer** [comment]
ReaInsert seems to be some REAPER specific plugin. I'm unable to open this in any other host.

Better to ask the REAPER dev if this should be possible. I don't think there is anything that we can do about this.
